panel.go

```diff
@@ -361,6 +353,80 @@ type (
 		YBucketNumber *float64 `json:"yBucketNumber"`
 		YBucketSize   *float64 `json:"yBucketSize"`
 	}
+	TimeseriesPanel struct {
+		Targets     []Target          `json:"targets,omitempty"`
+		Options     TimeseriesOptions `json:"options"`
+		FieldConfig FieldConfig       `json:"fieldConfig"`
+	}
+	TimeseriesOptions struct {
+		Legend  TimeseriesLegendOptions  `json:"legend,omitempty"`
+		Tooltip TimeseriesTooltipOptions `json:"tooltip,omitempty"`
+	}
+	TimeseriesLegendOptions struct {
+		Calcs       []string `json:"calcs"`
+		DisplayMode string   `json:"displayMode"`
+		Placement   string   `json:"placement"`
+	}
+	TimeseriesTooltipOptions struct {
+		Mode string `json:"mode"`
+	}
+	FieldConfigDefaults struct {
+		Unit       string            `json:"unit"`
+		Decimals   *int              `json:"decimals,omitempty"`
+		Min        *int              `json:"min,omitempty"`
+		Max        *int              `json:"max,omitempty"`
+		Color      FieldConfigColor  `json:"color"`
+		Thresholds Thresholds        `json:"thresholds"`
+		Custom     FieldConfigCustom `json:"custom"`
+		Links      []Link            `json:"links,omitempty"`
+	}
+	FieldConfigCustom struct {
+		AxisLabel         string `json:"axisLabel,omitempty"`
+		AxisPlacement     string `json:"axisPlacement"`
+		AxisSoftMin       *int   `json:"axisSoftMin,omitempty"`
+		AxisSoftMax       *int   `json:"axisSoftMax,omitempty"`
+		BarAlignment      int    `json:"barAlignment"`
+		DrawStyle         string `json:"drawStyle"`
+		FillOpacity       int    `json:"fillOpacity"`
+		GradientMode      string `json:"gradientMode"`
+		LineInterpolation string `json:"lineInterpolation"`
+		LineWidth         int    `json:"lineWidth"`
+		PointSize         int    `json:"pointSize"`
+		ShowPoints        string `json:"showPoints"`
+		SpanNulls         bool   `json:"spanNulls"`
+		HideFrom          struct {
```

Comment:
In an example dashboard I get:
```
            "hideFrom": {
              "graph": false,
              "legend": false,
              "tooltip": false
            },

```

On 7.5.1. Could you please point me to an example dashboard that has all of these settings? Or the typescript file?

Reply:
i see this in 8.3.1
```
        "hideFrom": {
          "tooltip": false,
          "viz": false,
          "legend": false
        },
```